DuckDuckGo/RootDebugViewController.swift

```diff
@@ -41,6 +42,7 @@ class RootDebugViewController: UITableViewController {
         case openVanillaBrowser = 670
         case resetSendCrashLogs = 671
         case refreshConfig = 672
+        case homeTabImprovements = 674
```

Comment:
I'm not overly keen on this.  Would be better to capture the cell as an outlet and check that. However, looks like we're invested in this approach.  We should probably refactor this debug view controller at some point. :) 

(No action required)